neuralmonkey/encoders/sentence_encoder.py

```diff
@@ -91,7 +91,7 @@ def __init__(self,
                 dtype=tf.float32)
 
             self.__attention_tensor = tf.concat(2, outputs_bidi_tup)
-            self.__attention_tensor = self._dropout(self._attention_tensor)
+            self.__attention_tensor = self._dropout(self.__attention_tensor)
```

Comment:
Co by neprošlo - akorát to šlo oklikou přes tu property místo toho, aby se volal rovnou ten atribut (měl jsem kliku, že to dělalo to samý, ale bylo to sémanticky blbě).